kalite/distributed/static/js/distributed/bundle_modules/register_public_key_client.js

```diff
@@ -0,0 +1,29 @@
+var $ = require("../base/jQuery");

```

Comment:
Not necessary to change, but worth noting that it is possible to use non-relative imports here as well (the build script collects all django app static/js/<appname> paths and makes them available).
```
var $ = require("base/jQuery");
```